Refs #25236 -- Removed ifequal/ifnotequal tags and docs.

timgraham opened the pull request:
https://code.djangoproject.com/ticket/25236

aaugustin commented:
After reviewing these changes, I'm thinking we should leave a short note in the docs. Otherwise, users who come across these tags in projets could have a hard time figuring out where they're coming from.

I'm thinking of something along the lines of: `{% ifequal a b %} ... {% endifequal %}` is an obsolete way to write `{% if a == b %} ... {% endif %}`. Likewise `{% ifnotequal a b %} ... {% endifnotequal %}` is superseded by `{% if a != b %} ... {% endif %}`

timgraham commented:
Sure, I don't mind doing that if the final decision is to postpone deprecation. While I was working on this, I thought of my latest proposal on the mailing list and I thought that might allow the deprecation to proceed. I added a second commit to show what code could be removed. It's 275 lines, mostly tests.

timgraham commented:
Removed usage in Django in a3830f6d663aee9e91871002cbff7c57aee6d989 and updated docs to discourage use in 787cc7aa843d5c0834f56b5f8ae03b86ce553c51. This PR is a reference of what could be removed if we deprecate these tags in the future.
